doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -38,7 +38,33 @@
 <a name="4.5"></a>
 ## 4.5 CNI plugins
 
-> This chapter should describe which CNI plugins are part of the Rerefence Architecture.
+The used CNI multiplexer/metapulgin must be [DANM](https://github.com/nokia/danm)
+as it provides the possibility to use several other CNI plugins (`req.inf.ntw.16`) and provides an API based solution to administer the networks (`req.inf.ntw.10`) from a central point (`req.inf.ntw.11`).<br>
+
+The following table contains a comparision of relevant features and requirements in Multus and DANM.
+
+| Requirement | Support in Multus | Support in DANM |
+|-------------|-------------------|-----------------|
+| `req.inf.ntw.01` | Supported | Supported |
+| `req.inf.ntw.02` | Supported via an other CNI plugin | Supported via an other CNI plugin |
+| `req.inf.ntw.03` | Supported via an other CNI plugin | Supported |
+| `req.inf.ntw.04` | Supported via an other CNI plugin | Supported via an other CNI plugin |
+| `req.inf.ntw.06` | Supported | Supported |
+| `req.inf.ntw.07` | Supported | Supported |
+| `req.inf.ntw.08` | Supported | Supported |
+| `req.inf.ntw.09` | Supported via LCM tools |  Supported via LCM tools |
+| `req.inf.ntw.10` | Not supported | Suported |
+| `req.inf.ntw.11` | Not supported | Partially supported |
+| `req.inf.ntw.14` | Supported via an other CNI plugin | Supported via an other CNI plugin |
+| `req.inf.ntw.15` | Not relevant | Not relevant |
+| `req.inf.ntw.16` | Supported | Supported |
+| Cluster wide IP address management | Not suported | Supported |
+| Service based dicovery of all provisioned interfaces | Not supported | Supported |
+
+ [Calico](https://github.com/projectcalico/cni-plugin) must be used as the CNI what complies with the basic networking assumptions of Kubernetes based on the requirement `req.inf.ntw.02` due to it's capability to handle `NetworkPolicies`, what is missing from [Flannel](https://github.com/coreos/flannel-cni).
+For the network of signalling connections the built in IPVLAN CNI of DANM must be used as it provides NAT-less connectivity (`req.inf.ntw.03`). For the user plane network(s) fullfilling requirement `req.inf.ntw.04` the [User Space CNI](https://github.com/intel/userspace-cni-network-plugin) must be used.
```

Comment:
IMO MACVLAN is as valid as IPVLAN, depends on whether CNF requires L2, or L3 unique addresses